src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -1614,6 +1614,14 @@
 helps['aks nodepool delete'] = """
 type: command
 short-summary: Delete the agent pool in the managed Kubernetes cluster.
+parameters:
+    - name: --ignore-pod-disruption-budget --ignore-pdb
```

Comment:
```suggestion
    - name: --ignore-pdb --ignore-pod-disruption-budget
```

Reply:
Yeah, it works now. Thanks a lot. 

I don't expect it is so strict for a parameter order in the helper file.